### PR TITLE
Refactored EC javascript and added table to EC monitor page

### PR DESCRIPTION
**Pull request by AlbertWhitlock**

### CODE REVIEW

**Comment by dlmarion** [comment]
@AlbertWhitlock  - I'm not sure what these changes do, can you provide a screenshot of the changes?

**Comment by DomGarguilo** [comment]
Here is a screenshot of the changed page:
image

The top table was added to replace what was described in #2934:
image

File: server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/ec.js [hunk 1]
```diff
@@ -192,35 +242,12 @@ function refresh() {
  * Generates the compactions table
  */
 function refreshECTables() {
```
Comment (DomGarguilo) [inline server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/ec.js:244]: I think the new table needs to be added to this method, I'm not seeing it refresh. Also, you can use `ajaxReloadTable()` from functions.js here.

**Comment by DomGarguilo** [comment]
Looks like an unrelated change was included in [009fe3d](https://github.com/apache/accumulo/pull/2997/commits/009fe3d4987ee7a8f4009fc7e9f3aac07b8d49e0), possibly by accident

**Comment by dlmarion** [comment]
Screenshots look good, I like it.

**Comment by DomGarguilo** [comment]
@AlbertWhitlock These changes look good. The only other suggestion I have is to use `Functions.ajaxReloadTable()` throughout this file to reload the tables. Other than that, I think this is good to go.

**Review by Manno15 — APPROVED** [review]
Changes look good once @DomGarguilo suggestion is taken care of.

**Review by DomGarguilo — APPROVED** [review]

**Pull request merged**